ynput/OpenPype
# Houdini: Workfile icon in new publisher

## Changelog Description

Fix icon for the workfile instance in new publisher

## Additional info

Before:
![afbeelding](https://user-images.githubusercontent.com/2439881/234024873-e4f8e671-6ba1-4050-8867-47d14e0798a2.png)

After:
![afbeelding](https://user-images.githubusercontent.com/2439881/234025178-92f305fc-3d2f-4653-ae25-1c426b4ac9b9.png)

## Testing notes:

1. Open Publisher
2. See the beautiful icon on the workfile instance
